miniapps/nurbs/nurbs_curveint.cpp

```diff
@@ -0,0 +1,238 @@
+// Compile with: make nurbs_curveint
+//
+// Sample runs:  ./nurbs_curveint -uw -n 9
+//               ./nurbs_curveint -nw -n 9
+
+//
+// Description:  This example code demonstrates the use of MFEM to interpolate
+//               a curve in a NURBS patch. We first define a square shaped
+//               NURBS patch. We then interpolate a sine function on the bottom
+//               edge. The results can be viewed in VisIt.
+//
+//               Us use curve interpolation for curves with all weights being 1,
```

Comment:
Us -> We

Reply:
Resolved in [e31bfca](https://github.com/mfem/mfem/pull/3232/commits/e31bfca43bf510dfc9e0cd51398ea06efa9707c6)